control-plane/agents/src/bin/core/pool/pool_operations.rs

```diff
@@ -36,6 +39,15 @@ impl ResourceLifecycle for OperationGuardArc<PoolSpec> {
             });
         }
 
+        if let Ok(pool) = registry.specs().pool(&request.id) {
```

Comment:
this validation is already done in start_create?

Reply:
Ah because if the node is not online we return that error instead of already exists..

Reply:
Its idempotent for Online Pool during operator update.